apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/implementation/redirect.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Redirect'
+description: 'Recipe for handling redirects using xmcloud and vercel.'
+hasSubPageNav: true
+hasInPageNav: false
+area: ['accelerate']
+---
+
+## Problem
+
+The customer has the following requirements:
+
+- R1. A marketer, strategist, or content author would like to create redirects.
+- R2. The Customer has a huge list of redirects after migration to xmcloud.
+
+## Solution for R1 - small number of redirects
+
+The xmcloud and nextjs app already have O.O.T.B. support for [Redirect](https://doc.sitecore.com/xmc/en/users/xm-cloud/redirect-a-url.html) and [Redirect Map](https://doc.sitecore.com/xmc/en/users/xm-cloud/create-a-redirect-map.html).
+Both solutions are quite useful as long as you do not have a large amount of redirects.
+
+## Solution for R2 - large number of redirects
+
+When you have a large number of redirects, you need to use the hosting provider features to handle it in your frontend application.
+
+For example, vercel has a few methods for handling redirects:
+
+- [Serverless functions](https://vercel.com/docs/edge-network/redirects#serverless-functions)
```

Comment:
These links can be moved to related documentation

Reply:
Is moved to the related documentation